net-libs/zeromq/metadata.xml

```diff
@@ -29,6 +36,9 @@
 			Build draft API, which may change at any time without any notice, and
 			is therefore not recommended for normal use.
 		</flag>
+		<flag name="intrinsics">
+			Build using compiler intrinsics for atomic ops.
```

Comment:
Although, I'm not sure if we want to make this a configure option. Or rather, it should be put behind `debug` use flag.

@thesamesam opinions?